mycore-base/src/main/java/org/mycore/common/config/MCRConfigurationDirSetup.java

```diff
@@ -54,10 +53,9 @@
  * @author Thomas Scheffler (yagee)
  * @since 2013.12
  */
+@SuppressWarnings("PMD.SystemPrintln")
 public class MCRConfigurationDirSetup implements AutoExecutable {
 
-    private static final StatusLogger LOGGER = StatusLogger.getLogger();
```

Comment:
Why not use the `StatusLogger` here (as in other classes that work in a pre-configured state of MyCoRe)? That seemed to be the better solution and is working fine.

Reply:
I just tested it and it doesn't work.

Reply:
Hm strange, I tested with an Ubuntu 24.04 and MIR 2024.06.01.
Apparently it also didn't work for @yagee-de back then, that's probably why the StatusLogger was just used once in the entire class before.